e2e-coho-tests/src/test/java/tests/QuiteModeToggleTest.java

```diff
@@ -0,0 +1,48 @@
+package Tests;
+
+import Pages.*;
+import Tests.AbstractBaseTests.TestBase;
+import cucumber.api.CucumberOptions;
+import cucumber.api.java.en.Given;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+
+@CucumberOptions(
+        strict = true,
+        monochrome = true,
+        features = "classpath:SmartWatchSettingsTest",
+        plugin = {"pretty"}
+)
+
+public class QuiteModeToggleTest extends TestBase {
+
+    private WelcomePage welcomePage;
+    private IntroductionPage introPage;
+    private GetStartedPage getStartedPage;
+    private SignInPage signInPage;
+    private DeviceDashboardPage deviceDashboardPage;
+    private ManageDevicesPage manageDevicePage;
+    private  CareSmartWatchSettingsPage careSmartWatchSettingsPage;
+
+
+    public String getName() {
+        return "Quite Mode Toggle Test";
+    }
+
+    @Given("^I navigate to the Care Smart Watch Settings Page")
+    public void setUpPage() {
+        welcomePage = new WelcomePage(driver);
+    }
+
+    @Given("^Enable Quite Mode$")
```

Comment:
for given steps it's better to have passive voice structure so it looks like a precondition.
```suggestion
    @Given("^The Quite Mode is enabled$")
```